### ISSUE TITLE
[BUG] In Windows using `--enable-themida` results in error `The system cannot find the path specified`

### ISSUE THREAD

**Issue by UnidentifiedCitizen** [dashingsoft/pyarmor#1299]
C:\Users\user\Desktop\1>pyarmor gen slave.py --enable-themida
INFO     Python 3.10.11
INFO     Pyarmor 8.2.4 (trial), 000000, non-profits
INFO     Platform windows.x86_64
INFO     search inputs ...
INFO     find script slave.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'windows.amd64'}
ERROR    [WinError 3] The system cannot find the path specified: 'C:\\Users\\user\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pyarmor\\cli\\core\\themida'

**Comment by jondy** [comment]
What's package `pyarmor.cli.runtime` version?

**Comment by jondy** [comment]
A quick workaround, apply this patch
```patch
+++ b/path/to/pyarmor/cli/core/__init__.py
@@ -152,6 +152,9 @@ class PyarmorRuntime(object):
         pkgpath = os_path.dirname(__file__)
 
         path = os_path.join(pkgpath, extra if extra else '')
+        if extra and not os_path.exists(path):
+            return
+
```

**Comment by jondy** [comment]
Fixed it in 8.2.5

**Comment by luigirovani** [comment]
there is a sintax error

use

`--enable themida
`
for exemplo

`pyarmor gen --enable themida __init__.py
`
